### PR TITLE
Add plugin chatwork notify

### PR DESCRIPTION
**Pull request by dieuhd**
## What does this change?
Allow push notification to Chatwork.com

Config chatwork-nofity:
```ts
{
  chatworkToken:"xxxx-x-xxxxxx";
  roomID:"123456789";
  mrUrl:"$CI_MERGE_REQUEST_PROJECT_URL/-/merge_requests/$CI_MERGE_REQUEST_IID"; // for gitlab
  mention:"[toall]";
  pipelineUrl:  "$CI_PIPELINE_URL";
}
```
## Screenshots
Screen Shot 2020-12-16 at 11 41 49 PM

### CODE REVIEW

File: packages/reg-notify-chatwork-plugin/src/chatwork-notifier-plugin.ts [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+
+import { sendWebHook } from "./send-web-hook";
+
+export interface ChatworkNotiferPluginOptions {
+  roomID: string;
+  chatworkToken: string;
+  mention?: string;
+  message?: string;
+  mrURL?: string;
+  pipelineURL?: string;
```
Comment (Quramy) [inline packages/reg-notify-chatwork-plugin/src/chatwork-notifier-plugin.ts:11]: [nits] reg-suit's other plugins use not `URL` but `Url` suffix convention (e.g. `webhookUrl` or `GitHubUrl` ).
Would you rename to `mrUrl` / `pipelineUrl` ?

File: packages/reg-notify-chatwork-plugin/src/chatwork-notifier-plugin.ts [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+      lines.push("");
+      lines.push(`■  Merge Request: ${this._mrURL}`);
+    }
+    if (this._pipelineURL && this._pipelineURL.length > 0) {
+      lines.push(`■  Pipeline URL: ${this._pipelineURL}`);
+    }
+    return lines.join("\n") + ` [/info]`;
+  }
+
+  createEmoji(params: NotifyParams) {
```
Comment (Quramy) [inline packages/reg-notify-chatwork-plugin/src/chatwork-notifier-plugin.ts:79]: I like this :)

File: packages/reg-notify-chatwork-plugin/src/chatwork-preparer.ts [hunk 3]
```diff
@@ -0,0 +1,56 @@
+import { PluginCreateOptions, PluginPreparer } from "reg-suit-interface";
+
+import { ChatworkNotiferPluginOptions } from "./chatwork-notifier-plugin";
+import { sendWebHook } from "./send-web-hook";
+
+export interface QuestionResult {
+  webhookUrl?: string;
```
Comment (Quramy) [inline packages/reg-notify-chatwork-plugin/src/chatwork-preparer.ts:7]: It seems that this property is never used. Please remove it.

**Review by Quramy — CHANGES_REQUESTED** [review]
@dieuhd Thanks for the PR !

I've add some comments. I'll merge soon if you fix them 😄

**Comment by dieuhd** [comment]
@Quramy Thanks for your comments.
I had just fixed it. Please review this PR again.

**Review by Quramy — APPROVED** [review]

**Comment by Quramy** [comment]
Thanks! I’ve published this as v0.10.10

**Pull request merged**